Source/core/CarbonFootprintCore.js

```diff
@@ -51,8 +51,8 @@ CarbonFootprintCore.KG_TO_LBS = 2.204622621848775;
  * @return {number} footprint
  */
 
-CarbonFootprintCore.prototype.computeFootprint = function(distance) {
-  var footprint = distance * this.settingsProvider.getCarbonEmission();
+CarbonFootprintCore.prototype.computeFootprint = function(quantity,type) {
```

Comment:
By renaming "distance" to "quantity" (which is now a number that can be a distance or an amount of time), the meaning of this function became less clear. It is only possible to understand what "quantity", "type" and hence the entire function mean when we look at the implementation of "getCarbonEmission"...

This is not ideal from a software engineering perspective. One should have to look at other parts of the code to understand what a function is doing.